src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -10,6 +10,55 @@
 
 namespace AppInstaller::Manifest
 {
+    namespace
+    {
+        std::map<AppInstaller::StringResource::StringId, std::string_view> ErrorIdToMessageMap = {
+            { AppInstaller::Manifest::ManifestError::InvalidRootNode, "Encountered unexpected root node."},

```

Comment:
Maybe mark all those literals as `sv` so they are `string_view`? (TBH, I'm not sure what difference it makes, I just know we do that in other places.)

Reply:
Depending on the context it might mean a `constexpr` construction of a view at compile time (with `sv`), or a runtime construction that needs to look for the null char.

But some contexts it doesn't matter.

Better to put it by default than to need to understand exactly when it will be one or the other.